### DISCUSSION TITLE
It is possible to eliminate the factory default color in Philips 9290019532

### DISCUSSION THREAD

**Discussion by kitopopo** [Koenkk/zigbee2mqtt#9469] [Devices]
Dear koenk friend, an idea has been around your head for a long time with philips hue bulbs. 
I would like to make a script and have them light up in the morning in a different color according to the outside temperature (cooler blue and hotter red), but I control these bulbs through a shelly and the wall switch cuts off the current. 
You know that philips has a default power-on color, it is a very ugly yellow, when they lose power the bulbs sometimes turn on in this default color. Can you think of any new ziggbee settings to eliminate this yellow color? I thought about using the option hue_power_on_behavior ":" off ", but when you cut the power to the bulbs, they would also light up yellow. Can you think of any improvements? Thanks in advance

**Comment by sjorge** [comment]
There is already a hue_power_on_color and hue_power_on_color_temperture that can be set to change the default color.

https://github.com/Koenkk/zigbee-herdsman-converters/blob/0df886ee834467bde0b7a10cdfa5c43e4c3f206d/converters/toZigbee.js#L1789

Is this not sufficient to do what you want?

**Reply by kitopopo** [reply]
> There is already a hue_power_on_color and hue_power_on_color_temperture that can be set to change the default color.
> 
> https://github.com/Koenkk/zigbee-herdsman-converters/blob/0df886ee834467bde0b7a10cdfa5c43e4c3f206d/converters/toZigbee.js#L1789
> 
> Is this not sufficient to do what you want?

Hi, thank you for you answer. I already know a hue_power_on_color and hue_power_on_color_temperture values ​​exist, but when the bulbs do not lose current and while we always turn them off using zigbee (without loss current). The problem is when we cut the power to the bulbs through a switch. In that case there are 3 possible turn on of the bulbs. 

1) that them turn on with the default philips color (this color ignores a hue_power_on_color and hue_power_on_color_temperture mode) 

2) that them turn on in the configured zigbee mode. 

3) if we keep the bulbs on for more than 10 seconds,  they can keep the mode configured by zigbee and the bulbs power on in in the color preset by zigbee (hue_power_on_color and hue_power_on_color_temperture mode)

When these bulbs lose current they behave in these 3 ways, in fact, if we turn off and turn on the bulbs in an interval of 1 or 2 seconds, the bulbs alternate between the default philips color and the one configured in zigbee in each 2 second ignition.

This behavior must come from factory or design, for this reason I ask if the firmware of the bulb could be retouched using some type of zigbee configuration or something similar.
